src/layouts/home/user-list-item/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Dispatch, SetStateAction } from 'react';
+
+import { ChatBriefViewResponse } from '@api/index';
+import { getDateDistance } from '@layouts/home/user-list-item/utils';
+import ChatAvatarSvg from '@public/icons/chat-avatar.svg';
+import GroupAvatarSvg from '@public/icons/group-avatar.svg';
+
+const userId = 'user001'; //temporary variable
+const unreadableMessages = 32; //temporary variable
+
+interface Props {
+  item: ChatBriefViewResponse;
+  setContentIsVisible: Dispatch<SetStateAction<boolean>>;
```

Comment:
It is bad practice to pass such a function directly. It is better to create a function in the parent component that will be executed in the child component